src/components/keyboard/keyboard.js

```diff
@@ -303,6 +303,15 @@ class Keyboard extends Component {
       this.props.notifyHoverChange({hover: false});
     }, CONTROL_BAR_HOVER_DEFAULT_TIMEOUT);
   }
+
+  /** get unhandled value for keyboard event
+   *
+   * @returns {KeyboardEventResult} - default result
+   * @memberof Keyboard
+   */
+  get unhandledKeyboardEventResult(): KeyboardEventResult {
+    return {preventDefault: false, payload: null};
```

Comment:
as it static const make it a `const unhandledKeyboardEventResult: KeyboardEventResult = {preventDefault: false, payload: null};`
place it in top of file and use it internally.